packages/web/hooks/ui-config/use-add-concentrated-liquidity-config.ts

```diff
@@ -53,22 +53,28 @@ export function useAddConcentratedLiquidityConfig(
   const addLiquidity = useCallback(async () => {
     return new Promise<void>(async (resolve, reject) => {
       try {
+        let quoteDepositValue;
+        let baseDepositValue;
+
+        if (!config.quoteDepositOnly) {
+          quoteDepositValue = {
+            currency: config.quoteDepositAmountIn.sendCurrency,
+            amount: config.quoteDepositAmountIn.amount,
+          };
+        }
+        if (!config.baseDepositOnly) {
+          baseDepositValue = {
+            currency: config.baseDepositAmountIn.sendCurrency,
+            amount: config.baseDepositAmountIn.amount,
+          };
+        }
```

Comment:
Nit: all of this code could be distilled into 2 ternary expressions... 

```
config.baseDepositOnly ? undefined : {
  currency: config.baseDepositAmountIn.sendCurrency,
  amount: config.baseDepositAmountIn.amount,
}
```

But not worth blocking this PR. Just FYI

Reply:
Sorry, I am shooting from the hip in terms of getting through ts syntax 😅

Reply:
https://github.com/osmosis-labs/osmosis-frontend/pull/1629/commits/71ed486f2badd135dd2a7ffec649b610095df2de